index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
+        <div class="navbar_menu">
+          <div class="navbar_menu_button">
+            <img src="assets/menu_icon.svg" alt="menu_icon">
+          </div>
+        </div>
+        
+      </div>
+      
+    </div>
+    <div class="main_section">
```

Comment:
- [ ] Kindly wrap your headline in a section tag as it represents a generic standalone section of a document. This will make your website semantic and enable search engines and screen readers understand the flow of your website 👎🏼